diff -urN licq-1.0.org/bin/Makefile.am licq-1.0/bin/Makefile.am
--- licq-1.0.org/bin/Makefile.am	Tue Oct 31 17:07:14 2000
+++ licq-1.0/bin/Makefile.am	Tue Oct 31 19:54:00 2000
@@ -7,11 +7,11 @@
 #TODO: convert this to something that works better
 #but for now it should be sufficient
 install-data-local:
-	$(mkinstalldirs) $(bindir)
+	$(mkinstalldirs) $(DESTDIR)$(bindir)
 	@for i in $(srcdir)/viewurl-*.sh; do \
 	  if test -f $$i; then \
-	    echo "$(INSTALL_PROGRAM) $$i $(bindir);"; \
-	    $(INSTALL_PROGRAM) $$i $(bindir); \
+	    echo "$(INSTALL_PROGRAM) $$i $(DESTDIR)$(bindir);"; \
+	    $(INSTALL_PROGRAM) $$i $(DESTDIR)$(bindir); \
 	  fi \
 	done
 
diff -urN licq-1.0.org/plugins/qt-gui-1.0/po/Makefile.am licq-1.0/plugins/qt-gui-1.0/po/Makefile.am
--- licq-1.0.org/plugins/qt-gui-1.0/po/Makefile.am	Tue Oct 31 17:07:16 2000
+++ licq-1.0/plugins/qt-gui-1.0/po/Makefile.am	Tue Oct 31 19:55:16 2000
@@ -34,21 +34,21 @@
 	done
 
 install-data-local:
-	$(mkinstalldirs) $(localedir)
+	$(mkinstalldirs) $(DESTDIR)$(localedir)
 	@for i in *.qm; do \
 	  if test "$$i" != "template.qm"; then \
-	    echo " $(INSTALL_DATA) $$i $(localedir)/$$i"; \
-	    $(INSTALL_DATA) $$i $(localedir)/$$i; \
+	    echo " $(INSTALL_DATA) $$i $(DESTDIR)$(localedir)/$$i"; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(localedir)/$$i; \
           fi \
 	done; \
-	cd $(localedir) && rm -f ru.qm; \
-	cd $(localedir) && rm -f cs.qm; \
-        cd $(localedir) && ln -s ru_RU.KOI8-R.qm ru.qm; \
-        cd $(localedir) && ln -s cs_CZ.qm cs.qm
+	cd $(DESTDIR)$(localedir) && rm -f ru.qm; \
+	cd $(DESTDIR)$(localedir) && rm -f cs.qm; \
+        cd $(DESTDIR)$(localedir) && ln -s ru_RU.KOI8-R.qm ru.qm; \
+        cd $(DESTDIR)$(localedir) && ln -s cs_CZ.qm cs.qm
 
 template: 
 	cd $(top_srcdir)/src && $(QT_FINDTR) *.cpp *.h > $(top_builddir)/po/template.po
 	cat addon.qt >>template.po
 
 uninstall-local:
-	-rm -f $(localedir)/*.qm
+	-rm -f $(DESTDIR)$(localedir)/*.qm
diff -urN licq-1.0.org/plugins/qt-gui-1.0/share/Makefile.am licq-1.0/plugins/qt-gui-1.0/share/Makefile.am
--- licq-1.0.org/plugins/qt-gui-1.0/share/Makefile.am	Tue Oct 31 17:07:16 2000
+++ licq-1.0/plugins/qt-gui-1.0/share/Makefile.am	Tue Oct 31 20:00:07 2000
@@ -10,44 +10,44 @@
 install-data-local:
 	@if test "$(KDEDIR)" != ""; then \
 	   if test -w "$(KDEDIR)"; then \
-	     $(mkinstalldirs) $(KDEDIR)/share/applnk/Internet; \
-	     $(INSTALL_DATA) licq.desktop $(KDEDIR)/share/applnk/Internet; \
+	     $(mkinstalldirs) $(DESTDIR)$(KDEDIR)/share/applnk/Internet; \
+	     $(INSTALL_DATA) licq.desktop $(DESTDIR)$(KDEDIR)/share/applnk/Internet; \
 	   else \
-	     $(mkinstalldirs) $(HOME)/.kde/share/applnk/Internet; \
-	     $(INSTALL_DATA) licq.desktop $(HOME)/.kde/share/applnk/Internet; \
+	     $(mkinstalldirs) $(DESTDIR)$(HOME)/.kde/share/applnk/Internet; \
+	     $(INSTALL_DATA) licq.desktop $(DESTDIR)$(HOME)/.kde/share/applnk/Internet; \
 	   fi  \
 	fi
-	$(mkinstalldirs) $(sharedir)
-	@chmod 0777 $(sharedir)
-	$(mkinstalldirs) $(sharedir)/qt-gui/locale
-	@chmod 0777 $(sharedir)/qt-gui/locale
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)
+	@chmod 0777 $(DESTDIR)$(sharedir)
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)/qt-gui/locale
+	@chmod 0777 $(DESTDIR)$(sharedir)/qt-gui/locale
 	@for j in qt-gui/skin.*; do \
 	  echo "Installing skin $$j"; \
-	  $(mkinstalldirs) $(sharedir)/$$j; \
-	  chmod 0777 $(sharedir)/$$j; \
+	  $(mkinstalldirs) $(DESTDIR)$(sharedir)/$$j; \
+	  chmod 0777 $(DESTDIR)$(sharedir)/$$j; \
 	  for i in $$j/*; do \
 			if test -f $$i; then \
-	      $(INSTALL_DATA) $$i $(sharedir)/$$j; \
+	      $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/$$j; \
 	    fi \
 	  done; \
 	done
 	@for j in qt-gui/dock.*; do \
 	  echo "Installing dock theme $$j"; \
-	  $(mkinstalldirs) $(sharedir)/$$j; \
-	  chmod 0777 $(sharedir)/$$j; \
+	  $(mkinstalldirs) $(DESTDIR)$(sharedir)/$$j; \
+	  chmod 0777 $(DESTDIR)$(sharedir)/$$j; \
 	  for i in $$j/*; do \
 			if test -f $$i; then \
-	      $(INSTALL_DATA) $$i $(sharedir)/$$j; \
+	      $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/$$j; \
 	    fi \
 	  done; \
 	done
 	@for j in qt-gui/icons.*; do \
 	  echo "Installing icons $$j"; \
-	  $(mkinstalldirs) $(sharedir)/$$j; \
-          chmod 0777 $(sharedir)/$$j; \
+	  $(mkinstalldirs) $(DESTDIR)$(sharedir)/$$j; \
+          chmod 0777 $(DESTDIR)$(sharedir)/$$j; \
 	  for i in $$j/*; do \
 			if test -f $$i; then \
-	      $(INSTALL_DATA) $$i $(sharedir)/$$j; \
+	      $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/$$j; \
 			fi \
 	  done; \
 	done
diff -urN licq-1.0.org/share/Makefile.am licq-1.0/share/Makefile.am
--- licq-1.0.org/share/Makefile.am	Tue Oct 31 17:07:14 2000
+++ licq-1.0/share/Makefile.am	Tue Oct 31 19:54:00 2000
@@ -9,42 +9,42 @@
 #TODO: convert this to something that works better
 #but for now it should be sufficient
 install-data-local:
-	$(mkinstalldirs) $(plugindir)
-	$(mkinstalldirs) $(sharedir)
-	$(mkinstalldirs) $(sharedir)/utilities
+	$(mkinstalldirs) $(DESTDIR)$(plugindir)
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)/utilities
 	@for i in $(srcdir)/utilities/*; do \
 	  if test -f $$i; then \
-	    echo "$(INSTALL_DATA) $$i $(sharedir)/utilities;"; \
-	    $(INSTALL_DATA) $$i $(sharedir)/utilities; \
+	    echo "$(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/utilities;"; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/utilities; \
 	  fi \
 	done
-	$(mkinstalldirs) $(sharedir)/translations
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)/translations
 	@for i in $(srcdir)/translations/*; do \
 	  if test -f $$i; then \
-	    echo "$(INSTALL_DATA) $$i $(sharedir)/translations;"; \
-	    $(INSTALL_DATA) $$i $(sharedir)/translations; \
+	    echo "$(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/translations;"; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/translations; \
 	  fi \
 	done
-	$(mkinstalldirs) $(sharedir)/sounds
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)/sounds
 	@for i in $(srcdir)/sounds/*.wav; do \
 	  if test -f $$i; then \
-	    echo "$(INSTALL_DATA) $$i $(sharedir)/sounds;"; \
-	    $(INSTALL_DATA) $$i $(sharedir)/sounds; \
+	    echo "$(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/sounds;"; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/sounds; \
 	  fi \
 	done
-	$(mkinstalldirs) $(sharedir)/sounds/fun
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)/sounds/fun
 	@for i in $(srcdir)/sounds/fun/*; do \
 	  if test -f $$i; then \
-	    echo "$(INSTALL_DATA) $$i $(sharedir)/sounds/fun;"; \
-	    $(INSTALL_DATA) $$i $(sharedir)/sounds/fun; \
+	    echo "$(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/sounds/fun;"; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/sounds/fun; \
 	  fi \
 	done
-	$(mkinstalldirs) $(sharedir)/sounds/icq
+	$(mkinstalldirs) $(DESTDIR)$(sharedir)/sounds/icq
 	@for i in $(srcdir)/sounds/icq/*; do \
 	  if test -f $$i; then \
-	    echo "$(INSTALL_DATA) $$i $(sharedir)/sounds/icq;"; \
-	    $(INSTALL_DATA) $$i $(sharedir)/sounds/icq; \
+	    echo "$(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/sounds/icq;"; \
+	    $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/sounds/icq; \
 	  fi \
 	done
-#       $(mkinstalldirs) $(mandir)
-#       $(INSTALL_DATA) licq.1 $(mandir)
+#       $(mkinstalldirs) $(DESTDIR)$(mandir)
+#       $(INSTALL_DATA) licq.1 $(DESTDIR)$(mandir)
