diff -urN licq-1.0.org/plugins/qt-gui-1.0/src/optionsdlg.cpp licq-1.0/plugins/qt-gui-1.0/src/optionsdlg.cpp
--- licq-1.0.org/plugins/qt-gui-1.0/src/optionsdlg.cpp	Tue Oct 31 17:07:17 2000
+++ licq-1.0/plugins/qt-gui-1.0/src/optionsdlg.cpp	Tue Oct 31 17:25:16 2000
@@ -440,7 +440,7 @@
     else
     {
       char szTransFileName[MAX_FILENAME_LEN];
-      snprintf(szTransFileName, MAX_FILENAME_LEN, "%s/%s/%s", BASE_DIR, TRANSLATION_DIR, QFile::encodeName(cmbTrans->currentText()).data());
+      snprintf(szTransFileName, MAX_FILENAME_LEN, "%s/%s/%s", SHARE_DIR, TRANSLATION_DIR, QFile::encodeName(cmbTrans->currentText()).data());
       gTranslator.setTranslationMap(szTransFileName);
     }
   }
