diff -urN licq-1.0.3.org/src/icqd-tcp.cpp licq-1.0.3/src/icqd-tcp.cpp
--- licq-1.0.3.org/src/icqd-tcp.cpp	Wed Mar 14 09:17:23 2001
+++ licq-1.0.3/src/icqd-tcp.cpp	Wed Mar 14 09:18:07 2001
@@ -1075,8 +1075,13 @@
         unsigned long back = 0xFFFFFF, fore = 0x000000;
         if (nInVersion <= 4)
           packet >> theSequence;
-        else
+        else {
           packet >> fore >> back;
+          if( fore == back ) {
+            back = 0xFFFFFF;
+            fore = 0x000000;
+          }
+        }
         packet >> licqChar >> licqVersion;
         nMask |= licqVersion;
         if (licqChar == 'L')
@@ -1148,7 +1153,14 @@
       {
         unsigned long back = 0xFFFFFF, fore = 0x000000;
         if (nInVersion <= 4) packet >> theSequence;
-        else packet >> fore >> back;
+        else {
+          packet >> fore >> back;
+          if(fore == back)
+          {
+            fore = 0x000000;
+            back = 0xFFFFFF;
+          }
+        }
         packet >> licqChar >> licqVersion;
         nMask |= licqVersion;
         if (licqChar == 'L')
@@ -1202,7 +1214,13 @@
       {
         unsigned long back = 0xFFFFFF, fore = 0x000000;
         if (nInVersion <= 4) packet >> theSequence;
-        else packet >> fore >> back;
+        else {
+          packet >> fore >> back;
+          if(fore == back) {
+            fore = 0x000000;
+            back = 0xFFFFFF;
+          }
+        }
         packet >> licqChar >> licqVersion;
         nMask |= licqVersion;
         if (licqChar == 'L')
