diff -urN licq-0.85.orig/plugins/qt-gui/src/optionsdlg.cpp licq-0.85/plugins/qt-gui/src/optionsdlg.cpp
--- licq-0.85.orig/plugins/qt-gui/src/optionsdlg.cpp	Thu Jun 29 18:56:54 2000
+++ licq-0.85/plugins/qt-gui/src/optionsdlg.cpp	Sat Jul 22 18:30:44 2000
@@ -436,7 +436,7 @@
     else
     {
       char szTransFileName[MAX_FILENAME_LEN];
-      sprintf(szTransFileName, "%s/%s/%s", BASE_DIR, TRANSLATION_DIR, QFile::encodeName(cmbTrans->currentText()).data());
+      sprintf(szTransFileName, "%s%s/%s", SHARE_DIR, TRANSLATION_DIR, QFile::encodeName(cmbTrans->currentText()).data());
       gTranslator.setTranslationMap(szTransFileName);
     }
   }
