diff -urN licq-20011027.org/plugins/qt-gui/share/Makefile.am licq-20011027/plugins/qt-gui/share/Makefile.am
--- licq-20011027.org/plugins/qt-gui/share/Makefile.am	Sat Oct 27 12:27:27 2001
+++ licq-20011027/plugins/qt-gui/share/Makefile.am	Sat Oct 27 13:31:48 2001
@@ -30,7 +30,7 @@
 	done
 	@for j in qt-gui/dock.*; do \
 	  echo "Installing dock theme $$j"; \
-	  $(mkinstalldirs) $(DESTOIR)$(sharedir)/$$j; \
+	  $(mkinstalldirs) $(DESTDIR)$(sharedir)/$$j; \
 	  for i in $$j/*; do \
 			if test -f $$i; then \
 	      $(INSTALL_DATA) $$i $(DESTDIR)$(sharedir)/$$j; \
