diff -Naur licq-0.85/plugins/console-0.30/src/Makefile.am licq-0.85-p/plugins/console-0.30/src/Makefile.am
--- licq-0.85/plugins/console-0.30/src/Makefile.am	Tue Apr 11 20:37:22 2000
+++ licq-0.85-p/plugins/console-0.30/src/Makefile.am	Sat Jun 10 15:07:17 2000
@@ -4,7 +4,7 @@
 licq_incdir = $(top_srcdir)/../../src/inc
 
 
-INCLUDES = -Wall -I$(srcdir) -I$(licq_incdir)
+INCLUDES = -Wall -I$(srcdir) -I$(licq_incdir) -I/usr/include/ncurses
 
 noinst_HEADERS = console.h event_data.h window.h
 
