--- licq-1.0.3/src/user.cpp.orig	Thu Mar 29 23:43:02 2001
+++ licq-1.0.3/src/user.cpp	Thu Mar 29 23:43:36 2001
@@ -943,9 +943,9 @@
     delete m_vcMessages[m_vcMessages.size() - 1];
     m_vcMessages.pop_back();
     decNumUserEvents();
-    if (gLicqDaemon != NULL)
-      gLicqDaemon->PushPluginSignal(new CICQSignal(SIGNAL_UPDATExUSER,
-       USER_EVENTS, m_nUin, nId));
+    // if (gLicqDaemon != NULL)
+    //  gLicqDaemon->PushPluginSignal(new CICQSignal(SIGNAL_UPDATExUSER,
+    //   USER_EVENTS, m_nUin, nId));
   }
 
   /* FIXME memory leak, but requires calling free, and that's a lot of typing
